### PR TITLE
Inline link colour

### LINKED ISSUE #367: Update InlineLink color to #222
**Is your feature request related to a problem? Please describe.**
Currently the InlineLink color is `#3F3F42` (same as body text), but the designs are `#222` (`C_EBON`).

**Describe the solution you'd like**
A PR to tweak the InlineLink color.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Cypress tests do now interact with this particular CSS therefore, updating the snapshot unit tests will suffice. 

**Additional context**
N/A

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by pjlee11**
Resolves #367

**Overall change:** _Changes the inlineLink colour from shadow to ebon_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-inline-link/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 | Version | Description |
 |---------|-------------|
+| 0.3.4   | [PR#323](https://github.com/bbc/psammead/pull/xxx) Update colour from Shadow to Ebon |
```
Comment (pjlee11) [inline packages/components/psammead-inline-link/CHANGELOG.md]:
```suggestion
| 0.3.4   | [PR#323](https://github.com/bbc/psammead/pull/371) Update colour from Shadow to Ebon |
```

File: packages/components/psammead-inline-link/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 | Version | Description |
 |---------|-------------|
+| 0.3.4   | [PR#323](https://github.com/bbc/psammead/pull/371) Update colour from Shadow to Ebon |
```
Comment (pjlee11) [inline packages/components/psammead-inline-link/CHANGELOG.md]:
```suggestion
| 0.3.4   | [PR#371](https://github.com/bbc/psammead/pull/371) Update colour from Shadow to Ebon |
```

**Review by ChrisBAshton — APPROVED** [review]
Looks good! 👍

**Review by dr3 — APPROVED** [review]

**Comment by staylos92** [comment]
LGTM 👍

**Pull request merged**